data/helpers.d/nodejs

```diff
@@ -126,6 +127,9 @@ ynh_install_nodejs () {
 
     # If n is not previously setup, install it
     if ! test $(n --version > /dev/null 2>&1)
+    then
+        ynh_install_n
+    elif dpkg --compare-version "$($n_install_dir/bin/n --version | cut -d" " -f2)" lt $n_version
```

Comment:
It's `--compare-versions` ;-)
And for me, there's no need to cut. Do you remember why you added it?

```suggestion
    elif dpkg --compare-versions "$($n_install_dir/bin/n --version)" lt $n_version
```

Reply:
aaah! one mistake per line 😱
I got it [here](https://github.com/YunoHost-Apps/Experimental_helpers/pull/52/files) (but the mistakes are mine...).
Thanks anyway for reviewing my crappy code.